src/utils/fetchClient.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-const BASE_URL = 'https://mate.academy/students-api';
+const BASE_URL = 'https://mate.academy/students-api/todos?userId=';
```

Comment:
```suggestion
const BASE_URL = 'https://mate.academy/students-api
```
it shouldn't be in the new function you should create (e.g. `loadTodos`)